designs/RestoreAndBuildCompatibility.md

```diff
@@ -156,23 +175,38 @@ Given that the implied compatibility exists, the proposal is for the SDK build t
 * Warn when the restore output is from an older tool than anticipated.
 * Warn when the restore output is from a newer tool than anticipated.
 
-There are 3 technical options and each might have a different effect on the frequency in which this warning is raised.
-
-#### Option 1 - Use assets file version
+### Assets file version
 
 NuGet increments the `project.assets.json` version every time there's a functional change in behavior. The build tasks in the SDK are built with a specific version of NuGet, and the SDK could raise a warning when the `project.assets.json` version is not the one it expects.
 
-Cons:
+This version that has historically been used for schema changes, not tooling compatibility checks. For example, aliases addition involved adding more information to the assets file, but not a schema change so this number was not incremented.
 
-* Thas version that has historically been used for schema changes, not tooling compatibility checks. For example, aliases addition involved adding more information to the assets file, but not a schema change so this number was not incremented.
+< TODO NK > Handle the complexity of an alternative .NET SDK?

```

Comment:
I look forward to this :)

Reply:
Me too, let me know when you figure something out :D